core/src/main/resources/error/error-classes.json

```diff
@@ -15,6 +15,10 @@
     "message" : [ "The second argument of '%s' function needs to be an integer." ],
     "sqlState" : "22023"
   },
+  "UNABLE_TO_ACQUIRE_MEMORY" : {
+    "message" : [ "Unable to acquire %s bytes of memory, got %s" ],
+    "sqlState" : null
```

Comment:
Can't we just omit the `sqlState` field?

Reply:
Yes; we just need to set the `JsonInclude` configuration. I updated the test to make sure this is empty if null.